packages/components/src/components/primitives/Popover/Popover.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { renderWithProviders, screen } from '@tests/renderWithProviders';
+import { Popover } from '.';
+import '@testing-library/jest-dom';
+
+describe('Popover', () => {
+  const givenComponentRendered = (isOpen = false) =>
+    renderWithProviders(
+      <Popover isOpen={isOpen}>
+        <Popover.Content data-testid="cmpsr.popover-content">Children</Popover.Content>
+      </Popover>
+    );
+
+  test('should render child if is open', () => {
+    givenComponentRendered(true);
+    expect(screen.queryByText(/Children/i)).toBeVisible();
```

Comment:
I decided to use `toBeVisible` instead of `toBeInTheDocument` since chakra doesn't remove the dom element, just changes the `visibility` attribute with css